jib-core/src/main/java/com/google/cloud/tools/jib/image/Image.java

```diff
@@ -79,11 +88,18 @@
   /** Initial command to run when running the image. */
   private final ImmutableList<String> entrypoint;
 
+  /** Arguments to pass into main when running the image. */
+  private final ImmutableList<String> cmd;
+
   private Image(
-      ImageLayers<T> layers, ImmutableList<String> environment, ImmutableList<String> entrypoint) {
+      ImageLayers<T> layers,
+      ImmutableList<String> environment,
+      ImmutableList<String> entrypoint,
+      ImmutableList<String> cmd) {
     this.layers = layers;
     this.environmentBuilder = environment;
     this.entrypoint = entrypoint;
+    this.cmd = cmd;
```

Comment:
We should use non-abbreviations for the `Image` class since it is a model and does not need to have the fields be as the same as in the JSON.

Reply:
"command" or "javaArguments"?